Linux-Device-Driver/3_Character_Driver_Multi-Devices/README.md

```diff
@@ -0,0 +1,89 @@
+<h1> Pseudo Character Driver which handles Multiple Device Instances </h1>
+
+## 1. Code exercise
+- Modify the previous pseudo character device driver to support four pseudo character devices.
+- Implement open, release, read, write, lseek driver methods to handle user requests.
+
+<p align="center"> <img width="750" src="https://user-images.githubusercontent.com/32474027/94355859-bed1ad80-00c2-11eb-9881-52bdc862f264.png" \></p>
+
+- The driver, which we about to write, should manage 4 devices.
+- There will be single driver managing 4 devices.
+- There will be only 1 implementation of open, read, write, release, and lseek driver methods.
+- That also means the driver should first determine which device is being accessed from the user space to fulfill the request
```

Comment:
Can we write more details about multi device instances.

Reply:
We explained it in the code in more details.